### DISCUSSION TITLE
posts/2024-07-09-t-cells/

### DISCUSSION THREAD

**Discussion by giscus** [racheltho/rachel-blog#14] [Announcements]
# posts/2024-07-09-t-cells/

Can AI tell us what a T cell will bind to? Part 1 of this series covers the basics of T cells and protein folding.

https://rachel.fast.ai/posts/2024-07-09-t-cells/

**Comment by joshpopelka20** [comment]
Thank you for writing this interesting article! Since AlphaFold came out, I've been curious how it would be used. I've dabbled in some online cell biology and biophysics courses. but none have really made it clear how AlphaFold could be used. This article does an excellent job explaining this (i.e. explaining some of the limitations of the model). Can't wait for Parts 2 and 3 to learn more :)

**Comment by Carlos-Chau** [comment]
Thank you Rachel for this series. 
I learnt Jeff Hammerbacher from you, and am looking to learn more from you!

**Comment by arjan-hada** [comment]
This dataset could be helpful:- https://www.kaggle.com/competitions/tcr-specificity-prediction-challenge/overview

**Reply by racheltho** [reply]
Thank you for sharing, I had not seen that dataset/competition!
